src/applications/static-pages/health-care-manage-benefits/get-medical-records-page/components/App/index.js

```diff
@@ -18,16 +27,24 @@ export const App = ({ isCernerPatient, showNewGetMedicalRecordsPage }) => {
     return <AuthContent />;
   }
 
+  if (hasFacilityException(facilityIDs, showAuthFacilityIDExceptions)) {
+    return <AuthContent />;
+  }
+
   return <UnauthContent />;
 };
```

Comment:
The data flow behind this is so confusing . The first condition, `showNewGetMedicalRecordsPage`, is easy because that's just our Flipper flag. The `isCernerPatient` and `hasFacilityException` distinction is what is getting me.

When we hardcoded the facility ID for the Columbus facility into the project, wasn't `isCernerPatient` returning as true for those users? Meaning that wouldn't users enrolled in the Columbus facility hit the `if (isCernerPatient) {` condition?

Reply:
I was thinking that the facility IDs per portal (like `showAuthFacilityIDExceptions` in this case) would replace the `CERNER_FACILITY_IDS` that we currently use in `src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js`, where we basically break down that single list of facility IDs into separate lists per portal

Reply:
The data flow is indeed VERY confusing...

So `isCernerPatient` is looking at `isCerner`, but not MPI's `isCerner` but our own custom one [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/user/selectors.js#L37). I _think_ we need to update this selector for Mann Grandstaff's launch but it probably requires some discussion between you + @jbalboni + @erikphansen + @sanlouise + myself 😬